src/course/en/course.json

```diff
@@ -39,6 +39,10 @@
         "_accessibility": {
             "skipNavigationText": "Skip navigation",
             "_ariaLabels": {
+                "answeredIncorrectly": "You answered incorrectly",
+                "answeredCorrectly": "You answered correctly",
+                "selectedAnswer": "selected",
+                "unselectedAnswer": "not selected",
```

Comment:
Do these properties require a corresponding change to `course.model.schema`?

Reply:
Done, hopefully

Reply:
done

Reply:
`_buttons.remainingAttemptsText` still needs aligning.

Also, wonder if we should add `_buttons.disabledAriaLabel` to course.json?

Reply:
i knew i'd checked it. but dyslexia killed me

Reply:
Sorry, this still isn't right. In https://github.com/adaptlearning/adapt_framework/pull/2255/commits/3e85179c95765a30fefaf842608ad07b5e9dfaeb the field titles were changed rather than the defaults.

Reply:
Resolved missing `disabledAriaLabel` with direct [commit](https://github.com/adaptlearning/adapt_framework/commit/28f761171c5db3d36e9e52b08ad8c4a6060ad00b) into course.json.

Reply:
Ta